src/components/DashboardMicroblockTransactionsTable.vue

```diff
@@ -1,74 +1,49 @@
 <template>
-  <paginated-content
-    :entities="transactions"
-    @prev-clicked="loadPrevTransactions"
-    @next-clicked="loadNextTransactions">
-    <table>
-      <thead>
-        <tr>
-          <th>Hash</th>
-          <th>Created</th>
-          <th>Type</th>
-          <th>Data</th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr
-          v-for="transaction in transactions.data"
-          :key="transaction.hash ">
-          <td>
-            <hash-symbol>
-              th
-            </hash-symbol>
-
-            <value-hash-ellipsed
-              :hash="transaction.hash"
-              :link-to="`/transactions/${transaction.hash}`"/>
-          </td>
-          <td>
-            <datetime-label :datetime="transaction.created"/>
-          </td>
-          <td>{{ transaction.type }}</td>
-          <td>
-            <transaction-cell
-              :transaction-data="transaction.data"
-              :transaction-type="transaction.type"/>
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </paginated-content>
+  <table>
+    <thead>
+      <tr>
+        <th>Hash</th>
+        <th>Created</th>
+        <th>Type</th>
+        <th>Data</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr
+        v-for="transaction in transactions.data"
+        :key="transaction.hash ">
+        <td>
+          <hash-symbol>
+            th
+          </hash-symbol>
+          <value-hash-ellipsed
+            :hash="transaction.hash"
+            :link-to="`/transactions/${transaction.hash}`"/>
+        </td>
+        <td>
+          <datetime-label :datetime="transaction.created"/>
+        </td>
+        <td>{{ transaction.type }}</td>
+        <td>
+          <transaction-cell
+            :transaction-data="transaction.data"
+            :transaction-type="transaction.type"/>
+        </td>
+      </tr>
+    </tbody>
+  </table>
 </template>
+
 <script setup>
 import HashSymbol from '@/components/HashSymbol'
-import PaginatedContent from '@/components/PaginatedContent'
 import DatetimeLabel from '@/components/DatetimeLabel'
-import { useRecentBlocksStore } from '@/stores/recentBlocks'
 import TransactionCell from '@/components/TransactionCell'
 import ValueHashEllipsed from '@/components/ValueHashEllipsed'
 
-const { fetchSelectedMicroblockTransactions } = useRecentBlocksStore()
-
-const props = defineProps({
+defineProps({
   transactions: {
     required: true,
     type: Object,
   },
 })
-
-function loadPrevTransactions() {
-  fetchSelectedMicroblockTransactions(props.transactions.prev)
```

Comment:
I guess the param is not needed anymore in the function definition

Reply:
query params does not seems to be used anywhere in the app
![image](https://github.com/aeternity/aescan/assets/15363559/c6490e1c-a371-41f0-ad90-e37b3fcb3152)

Reply:
Fixed in the latest commit